fitting/views.py

```
@@ -25,11 +25,14 @@
 import logging
 from django.utils import timezone
 import datetime
+from google import genai
+import google.generativeai as genai
```

Ruff (0.12.2)

Redefinition of unused `genai` from line 28

(F811)